blake2b.h

```diff
@@ -228,7 +255,6 @@ int blake2b_init_param(blake2b_state *S, const blake2b_param *P) {
   return 0;
 }
 
-const char *DEFAULT_PERSONAL = "ckb-default-hash";
 int blake2b_init(blake2b_state *S, size_t outlen) {
```

Comment:
This function is still using `const char *DEFAULT_PERSONAL = "ckb-default-hash"`

Reply:
This is for compatibility reasons, we have existing code that relies on this function, it might not make sense to introduce a breaking change here. In an ideal world we will revert the code but for now I feel we might have to keep it like this.